include/SFML/Network/Ftp.h

```diff
@@ -506,5 +506,25 @@ CSFML_NETWORK_API sfFtpResponse* sfFtp_download(sfFtp* ftp, const char* distantF
 ////////////////////////////////////////////////////////////
 CSFML_NETWORK_API sfFtpResponse* sfFtp_upload(sfFtp* ftp, const char* localFile, const char* destPath, sfFtpTransferMode mode);
 
+////////////////////////////////////////////////////////////
+/// \brief Send a command to the FTP server
+///
+/// While the most often used commands are provided as member
+/// functions in the sfFtp class, this method can be used
+/// to send any FTP command to the server. If the command
+/// requires one or more parameters, they can be specified
+/// in \a parameter. Otherwise it should be an empty string.
```

Comment:
"Otherwise it should be an empty string" should probably read "Otherwise you should pass NULL".